docs/app/index.ejs

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <!-- baseHref value already includes quotes "/foo" -->
   <base href=<%= htmlWebpackPlugin.options.baseHref %> />
-  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/<%= htmlWebpackPlugin.options.versions.highlightjs %>/styles/github.min.css">
+  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/github-gist.min.css">
```

Comment:
We use react-highlight for syntax highlighting.  We'll replace it some day with a live editor.  For now, we manually update the highlight CSS version here.